src/pages/MarketDetails/Card/index.tsx

```diff
@@ -16,13 +16,20 @@ export interface ICardProps {
   legends?: ILegend[];
   stats?: IStat[];
   className?: string;
+  'data-testid'?: string;
```

Comment:
I think the alternative of keeping all props camelCased and using them ass attributes inline is preferable

Reply:
Works for me